### PR TITLE
[9.0] move new V8 module: stock_quant_manual_assign

### PR DESCRIPTION
**Pull request by cyrilgdn**
stock_quant_manual_assign is V8 module which was not moved in V9.

### CODE REVIEW

File: stock_quant_manual_assign/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Mikel Arregi - AvanzOSC
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    "name": "Stock - Manual assignment of quants",
+    "version": "8.0.1.0.0",
```
Comment (pedrobaeza) [inline stock_quant_manual_assign/__openerp__.py:7]: Change version number

**Comment by pedrobaeza** [comment]
Is this finished, because the module remains uninstallable?

File: stock_quant_manual_assign/__openerp__.py (hunk 1 above)
Comment (cyrilgdn) [inline stock_quant_manual_assign/__openerp__.py:7]: @pedrobaeza Sorry, my PR message is not very clear. I'm just adding this V8 module as unported  in V9 (i will create similar PR for stock_quant_merge and partner_location_auto_create).

We are merging, with @gurneyalex, 8.0 into 9.0 (splitted by module).

Comment (pedrobaeza) [inline stock_quant_manual_assign/__openerp__.py:7]: OK, understood. Then it's correct, except you can omit the line in the README.md, that is going to be changed in the next automated cron that scans modules.

Comment (cyrilgdn) [inline stock_quant_manual_assign/__openerp__.py:7]: Ok :)  I did not know it was automatic, thanks!

**Comment by pedrobaeza** [comment]
:+1:

**Comment by gurneyalex** [comment]
:+1:

**Pull request merged**